aiozipkin/aiohttp_helpers.py

```diff
@@ -36,6 +36,12 @@ def middleware_maker(tracer_key=APP_AIOZIPKIN_KEY,
                 span.tag(HTTP_PATH, request.path)
                 span.tag(HTTP_METHOD, request.method.upper())
 
+                peername = request.transport.get_extra_info('peername')
```

Comment:
Most likely in prod config `peername` points to reverse proxy server.
That's why `request.remote` exists.

Reply:
If reverse proxy (e.g. NGINX) is configures to use UNIX sockets for backends -- the name is a socket path, not host/port pair.
For IPv6 `peername` contains more than two elements, tail should be stripped.